package.json

```diff
@@ -29,11 +29,19 @@
   },
   "devDependencies": {
     "babel-eslint": "^7.2.3",
+    "chai": "^4.1.0",
+    "chai-as-promised": "^7.1.1",
     "eslint": "^4.2.0",
+    "jsdom": "^11.1.0",
+    "mocha": "^3.4.2",
```

Comment:
Question on running tests: I've got no `mocha` binary with a standard `yarn install`. What's the best way to ensure this shows up so I can run tests, locally and in Docker? I've exported `NODE_ENV=development` locally but `yarn install` still installs no `devDependencies`.